docs/templates.md

```diff
@@ -401,6 +401,9 @@ practices:
   - The Coder agent logs are typically stored in `/tmp/coder-agent.log`
   - The Coder agent startup script logs are typically stored in
     `/tmp/coder-startup-script.log`
+- This can also happen if the websockets are not being forwarded correctly when running Coder behind a reverse proxy. Follow our examples.
+  - [NGINX](https://github.com/coder/coder/tree/main/examples/web-server/nginx)
+  - [Apache](https://github.com/coder/coder/tree/main/examples/web-server/apache)
```

Comment:
Can you change this to a generic link to our reverse proxy docs? If we add more examples (e.g. Traefik), it'd suck if we forget to add them to both places?

```suggestion
- This can also happen if the websockets are not being forwarded correctly when running Coder behind a reverse proxy. [Read our reverse-proxy docs](https://coder.com/docs/v2/latest/admin/configure#tls--reverse-proxy)
```

We could also show the basic headers that need to be set on that page.

Reply:
Sure. I agree. Adding required configuration headers will make that over congested.